source/_components/sensor.miflora.markdown

```diff
@@ -13,51 +13,25 @@ ha_release: 0.29
 ha_iot_class: "Local Polling"
 ---
 
-The `miflora` sensor platform allows one to monitor plants. The [Mi Flora plant sensor](https://xiaomi-mi.com/sockets-and-sensors/xiaomi-huahuacaocao-flower-care-smart-monitor/) is a small Bluetooth Low Energy device that monitors not only the moisture but also light, temperature, and conductivity. As only a single BLE device can be polled at the same time, the library implements locking to make sure this is the case.
-
-# Installation
-Depending on the operating system you're running, you have to configure the proper Bluetooth backend on your system:
-
-- On [Hass.io](/hassio/installation/): Miflora will work out of the box.
-- On a [generic Docker installation](/docs/installation/docker/): Works out of the box with `--net=host` and properly configured Bluetooth on the host.
-- On other Linux systems:
-    - Preferred solution: Install the `bluepy` library (via pip). When using a virtual environment, make sure to use install the library in the right one.
-    - Fallback solution: Install `gatttool` via your package manager. Depending on the distribution, the package name might be: `bluez`, `bluetooth`, `bluez-deprecated`
-- Windows and MacOS are currently not supported by the [miflora library](https://github.com/open-homeautomation/miflora/).
-
-# Configuration
-Start a scan to determine the MAC addresses of the sensor:
-
-```bash
-$ sudo hcitool lescan
-LE Scan ...
-F8:04:33:AF:AB:A2 [TV] UE48JU6580
-C4:D3:8C:12:4C:57 Flower mate
-[...]
-```
-
-Or if your distribution is using bluetoothctl:
-
-```bash
-$ bluetoothctl
-[bluetooth]# scan on
-[NEW] Controller <your Bluetooth adapter> [default]
-[NEW] F8:04:33:AF:AB:A2 [TV] UE48JU6580
-[NEW] C4:D3:8C:12:4C:57 Flower mate
-```
-
-
-Check for `Flower care` or `Flower mate` entries, those are your sensor.
+The `miflora` sensor platform allows one to monitor plant soil and air conditions. The [Mi Flora plant sensor](https://xiaomi-mi.com/sockets-and-sensors/xiaomi-huahuacaocao-flower-care-smart-monitor/) is a small Bluetooth Low Energy device that monitors the moisture and conductivity of the soil as well as ambient light and temperature. Since only one BLE device can be polled at a time, the library implements locking to prevent polling more than one device at a time.
 
+# Configure
 To use your Mi Flora plant sensor in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 sensor:
   - platform: miflora
     mac: 'xx:xx:xx:xx:xx:xx'
+    name: Flower 1
+    force_update: false
+    median: 3
     monitored_conditions:
+      - moisture
+      - light
       - temperature
+      - conductivity
+      - battery
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps users to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.